SpatialGDK/Source/SpatialGDKTests/SpatialGDK/Interop/SpatialWorkerFlags/SpatialWorkerFlagsTest.cpp

```diff
@@ -0,0 +1,74 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+#include "Interop/SpatialWorkerFlags.h"
```

Comment:
is there no need for a headerfile for the test files?

Reply:
What header file would that be?